automation/packaging/rpm/el10/run-build.sh

```
@@ -0,0 +1,23 @@
+if [ -z "$REL" ]; then
+        RELEASE="el10"
+else
+        RELEASE="$REL.el10"
+fi
+sed -i "s/%RELEASE%/$RELEASE/g" jans.spec
+rpmbuild_path="$current_dir/rpmbuild"
+mkdir -p "$rpmbuild_path"/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
+specfile=jans.spec
+cp "$current_dir"/$specfile "$rpmbuild_path"/SPECS/.
+mv jans-src jans-"$VERSION"
+tar cvfz jans-"$VERSION".tar.gz jans-$VERSION
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)